mobile/src/bundles/common/components/slider/slider.tsx

```diff
@@ -0,0 +1,60 @@
+import CommunitySlider, {
+    type SliderProps,
+} from '@react-native-community/slider';
+import React from 'react';
+import { type StyleProp, type ViewStyle } from 'react-native';
+
+import { Text, View } from '~/bundles/common/components/components';
+import { Color, TextCategory } from '~/bundles/common/enums/enums';
+import { useWindowDimensions } from '~/bundles/common/hooks/hooks';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { styles } from './styles';
+
+type SliderProperties = {
+    thumbTitleValue: string;
+    containerStyle?: StyleProp<ViewStyle>;
+    value: number;
+    thumbTitleValueWidth?: number;
```

Comment:
Please put optional always as last 
```suggestion
    thumbTitleValue: string;
     value: number;
     thumbTitleValueWidth?: number;
    containerStyle?: StyleProp<ViewStyle>;
```

Reply:
fix